Guided Investigation - Incident Triage.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    resps = pd.DataFrame()
</pre><p>I really don't like variable names like this. :-)</p><p>I was trying wondering why nothing was appearing in the next cell and nothing about "resps" told me where it might have come from.</p><p>maybe "ip_ti_lookup_results" ?</p><p>I think screen space normally not expensive - giving vars descriptive names helps so much when you're trying to understand what a variable is about. Otherwise you have to trawl back and try to find the context where it was populated.</p><p>(Excuse the rant!)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>